datasets/haalcentraal/brk/dataset.json

```diff
@@ -1,12 +1,11 @@
 {
   "type": "dataset",
-  "id": "hcbrk",
+  "id": "haalcentraal_brk",
   "title": "BRK van HaalCentraal",
   "status": "beschikbaar",
   "description": "Deze dataset maakt de HaalCentraal BRK API toegankelijk, bij gebruik van login credentials. De velden zijn gebaseerd op het schema van Haal-Centraal.",
   "version": "0.0.1",
   "crs": "EPSG:28992",
-  "auth": "BRK/RSN",
   "tables": [
     {
       "id": "kadastraalonroerendezaken",
```

Comment:
Shouldn't there be an `identifier` for the tabel (as a (compound) PK)? Or, is that not neccesary for a virtual dataset?

Reply:
Added.